packages/button/test/lion-button.test.js

```diff
@@ -117,6 +117,8 @@ describe('lion-button', () => {
       `);
 
       pressSpace(form.querySelector('lion-button'));
+      await aTimeout();
+      await aTimeout();
```

Comment:
Although this is not nice, with the current implementation of helpers from `@polymer/iron-test-helpers/mock-interactions.js` there is no other way to determine the moment when `pressSpace/pressEnter` finish their job.